schema/cassandra/temporal/versioned/v1.6/executions.cql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE executions ADD tiered_storage_task_data blob;
+ALTER TABLE executions ADD tiered_storage_task_encoding text;
```

Comment:
should also update schema.cql, same for SQL schema files

Reply:
Done